Sources/Structs/Regex.swift

```diff
@@ -0,0 +1,232 @@
+//
+//  Regex.swift
+//  HandySwift
+//
+//  Created by Frederick Pietschmann on 19.03.18.
+//  Copyright © 2018 Flinesoft. All rights reserved.
+//
+// Taken from https://github.com/sharplet/Regex, then modified to remove some weight.
```

Comment:
Use `Originally from: https://...` instead. Also place a second whitespace at the beginning to allign with the above.